ecosystem/sep-0041.md

```diff
@@ -200,6 +202,12 @@ The event has topics:
 The event has data:
 
 - `i128` the amount transferred.
+- or `map` containing entries:
+  - `amount: i128` the amount minted.
+  - `to_muxed_id: Option<u64 | String | BytesN<32>>` void or absent if no muxed
```

Comment:
Have specified that the field can be absent to make this automatically compatible with:
- https://github.com/orgs/stellar/discussions/1750